app/lib/actions/katello/content_view/publish.rb

```diff
@@ -70,6 +70,12 @@ def humanized_name
         end
 
         def run
+          # update errata applicability counts for all hosts in the CV & Library
+          ::Katello::Host::ContentFacet.where(:content_view_id => input[:content_view_id],
+                                              :lifecycle_environment_id => library.id).each do |facet|
```

Comment:
`library` isn't defined here

If you move this block a few lines down you can do `view.organization.library.id`